websites/S/Starz/dist/metadata.json

```diff
@@ -3,12 +3,18 @@
     "name": "Bryan",
     "id": "293775922757238784"
   },
+  "contributors": [
+    {
+      "name": "Kedi",
+      "id": "238668511222693888"
+    }
+  ],
```

Comment:
If you want to you can make yourself author and bryan contributor. Since Bryan doesn't help out anymore 😢

Reply:
Rest in peace Bryan